images/arm-img-builder.sh

```diff
@@ -216,12 +216,6 @@ if [ -z "$output_image" ]; then
 fi
 
 if [ -n "$manifest" ]; then
-  YQ_VERSION=$( yq -V | cut -d " " -f 3 | cut -d "." -f 1)
```

Comment:
mm this seems to have broken the CI here: https://github.com/c3os-io/c3os/runs/6760298289?check_suite_focus=true#step:11:150